frontend/src/languages/fr.json

```diff
@@ -20,21 +21,18 @@
   "from.title": "De",
   "to.title": "À",
   "report.label.site.dateType": "Type de date",
-  "banner.menu.reports": "Rapports",
```

Comment:
hey @Lovelyfin00 why was this line removed? what issue was it causing?

Reply:
@parthnagdev 
It was duplicated
Removed it because it showed up twice on the file

Reply:
@parthnagdev 
Sorry, I just cross checked
I thought it was duplicated as a result of merge conflicts.
I can add it back right now

Reply:
I opened a [PR for this ](https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/886)